README.md

```diff
@@ -25,6 +25,9 @@ We recommend you start using the GDK if:
 
 Although every feature you need to build your optimal SpatialOS game may not be ready yet, this is a great time to get involved and shape the Unreal GDK. We are committed to improving the GDK rapidly, aiming for an alpha release in Autumn 2018.
 
+## Feature List
+  See the [full feature list](docs/features.md) for what is supported in the Unreal GDK
+
```

Comment:
Thank you. Not sure how we missed this off!

As you are in there, would you jiggle the page order slightly?

Reply:
How would you like it jiggled